### DISCUSSION TITLE
Malformed WAV file

### DISCUSSION THREAD

**Discussion by rousseldenis** [GrandOrgue/grandorgue#1765] [Developers]
@oleg68 I have a problem with sample set Prytanee (full).

A wav file seems to not pass the check you introduced there (check_for_bounds() function): https://github.com/GrandOrgue/grandorgue/commit/8a8e13bdd956ea6c8000903b8d2a79455efb8475

I don't know how to fix this as the wav file seems to be well read in standard sound files readers (like VLC). For obvious reasons I cannot post the file here.

Isn't it too restrictive ?

**Comment by oleg68** [comment]
Please post here the exact error message with actual numbers.
